File: GTTS/INL.VettingService.Models/GetInvestVettingBatch_Result.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class GetInvestVettingBatch_Result : BaseResult, IGetInvestVettingBatch_Result
    {
        public List<InvestVettingBatch_Item> InvestVettingBatchItems { get; set; }
        public int FileVersion { get; set; }
        public string FileName { get; set; }
        public int FileSize { get; set; }
        public byte[] FileHash { get; set; }
        public byte[] FileContent { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-invest-vetting-batch_result.ts


import { InvestVettingBatch_Item } from './invest-vetting-batch_item';
import { IGetInvestVettingBatch_Result } from './iget-invest-vetting-batch_result';

export class GetInvestVettingBatch_Result implements IGetInvestVettingBatch_Result {
  
	public InvestVettingBatchItems?: InvestVettingBatch_Item[];
	public FileVersion: number = 0;
	public FileName: string = "";
	public FileSize: number = 0;
	public FileHash: number[] = null;
	public FileContent: number[] = null;
  
	public ErrorMessages: string[] = null;
}


